packages/ocamlsdl2/ocamlsdl2.0.02/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+authors: [
+  "Florent Monnier"
+  "Piotr Mardziel"
+  "David Cadé"
+]
+license: "restrictionless Zlib"
+homepage: "https://github.com/fccm/OCamlSDL2"
+bug-reports: "https://github.com/fccm/OCamlSDL2/issues"
+dev-repo: "git+https://github.com/fccm/OCamlSDL2.git"
+doc: "http://www.linux-nantes.org/~fmonnier/ocaml/sdl2/doc/"
+tags: [
+  "bindings" "graphics" "audio" "multimedia" "opengl"
+  "cross-platform"
+]
+synopsis: "Interface to the SDL2 library"
+description: """
+An OCaml interface to the SDL2 multimedia library for Linux and Windows."""
+depends: [
+  "ocaml"
+  "ocamlfind"
```

Comment:
From a brief look at the code, it seems to me that `ocamlfind` is only used as `{build}` dependency